Northstar.CustomServers/mod.json

```diff
@@ -117,6 +117,14 @@
 		{
 			"Path": "lobby/sh_private_lobby_modes_init.gnut",
 			"RunOn": "( SERVER || CLIENT ) && MP"
+		},
+
+		{
+			"Path": "_respawned_protect.gnut",
```

Comment:
I think a better name for this feature would be 'Respawn Protection', can you update please?

Reply:
done